client/components/App.jsx

```diff
@@ -1,33 +1,124 @@
 import React from 'react';
-import TopComponent from './TopComponent.jsx'
+import TopComponent from './topComponent.jsx';
```

Comment:
Not sure of the exact context, but perhaps the "TopComponent" and "BottomComponent" could be a little more specific with their names